src/modules/app/containers/I18nRedirect.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { replace } from 'redux-little-router'
+import normalizeUrl from 'normalize-url'
+
+import { connect } from 'react-redux'
+
+export class I18nRedirect extends React.Component {
```

Comment:
please write a short documentation what your classes are used for, I18nRedirect for example is just used to redirect if we don't have a language in our path right?